### DISCUSSION TITLE
Availability of the Real Time Listings Feed

### DISCUSSION THREAD

**Discussion by liamcrayden** [etsy/open-api#872] [Ideas]
It seems we still have no definitive answer on this and we are only a month away from the V2 sunset date.

Will the existing real time listings feed continue to function past this date? It's not listed as retired, but as far as I know it's not being actively worked on either.

https://www.etsy.com/developers/documentation/getting_started/real_time_feeds

**Comment by Gareth064** [comment]
No feeds in V3

**Reply by JoannSellerServices** [reply]
Etsy should Hire you to be their community manager, lol.  You are the most helpful / responsive community advocate I've seen here.

**Reply by Gareth064** [reply]
I've worked in IT support for nearly 12 years. Specifically in the in Applications space. Its basically second nature for me to help people when I can, and I like to :)

**Reply by indiemade** [reply]
To make things more confusing the V2 documentation is still online and probably shouldn't be at such a high-profile URL.

https://www.etsy.com/developers/documentation/

If Gareth is correct Real-Time feeds and more importantly to us, the PubSubHubbub feeds have been retired.
